From 5d6dd78232716dbec4c4b11bad897beb8f13341e Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 11 Jan 2024 16:41:27 +0100
Subject: [PATCH 0428/1695] plat-stm32mp2: Enable regulator support

Enables regulator support on stm32mp2 platform and default
enable CFG_DRIVERS_REGULATOR_PRINT_TREE.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 3765e5e010181f2ebb888ae712677096124b30e2 plat-stm32mp2: Enable regulator support]
Change-Id: I23d25ce278293d7fed743dc2c23d79a5c9706372
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 30405c2b6..eb40b26a5 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -41,6 +41,7 @@ $(call force,CFG_DRIVERS_CLK,y)
 $(call force,CFG_DRIVERS_CLK_DT,y)
 $(call force,CFG_DRIVERS_GPIO,y)
 $(call force,CFG_DRIVERS_PINCTRL,y)
+$(call force,CFG_DRIVERS_REGULATOR,y)
 $(call force,CFG_DT,y)
 $(call force,CFG_GIC,y)
 $(call force,CFG_HALT_CORES_ON_PANIC_SGI,15)
@@ -91,7 +92,11 @@ CFG_STM32_STGEN ?= y
 CFG_STM32_TAMP ?= y
 CFG_STM32_UART ?= y
 
+CFG_REGULATOR_FIXED ?= y
+CFG_REGULATOR_GPIO ?= y
+
 # Default enable some test facitilites
+CFG_DRIVERS_REGULATOR_PRINT_TREE ?= y
 CFG_ENABLE_EMBEDDED_TESTS ?= y
 CFG_WITH_STATS ?= y
 CFG_WERROR ?= y
-- 
2.47.2

